x-pack/legacy/plugins/apm/scripts/optimize-tsconfig/optimize.js

```diff
@@ -50,32 +48,37 @@ function updateParentTsConfigs() {
   );
 }
 
+async function addApmFilesToXpackTsConfig() {
+  const template = json5.parse(await readFile(tsconfigTpl, 'utf-8'));
+  const xpackTsConfig = path.join(xpackRoot, 'tsconfig.json');
+  const config = json5.parse(await readFile(xpackTsConfig, 'utf-8'));
+
+  await writeFile(
+    xpackTsConfig,
+    JSON.stringify(assign(config, template), null, 2),
```

Comment:
whoa, haven't seen `_.assign` used since the ancient es5 times :D I must admit, I thought it was only a legacy `Object.assign`. 


Would this not work?
```suggestion
    JSON.stringify({...config, ...template}, null, 2),
```

Reply:
aye, I was changing it from `merge`, so lodash was still living in my head 😬 will change.